src/main/resources/assets/js/components/TaskListItemComponent.jsx

```diff
@@ -117,9 +134,13 @@ define([
             </span>
           </td>
           <td className="text-right">
-            <time dateTime={task.get("version").toISOString()} title={task.get("version").toISOString()}>
-              {task.get("version").toLocaleString()}
-            </time>
+            <span
+              title={task.get("version").toISOString()}
+              className={versionClassSet}
+              onClick={this.handleVersionClick}>{
+                this.state.versionShowTimestamp ?
```

Comment:
I've talked with @drexin and removed the click interaction. It's enough to see the older versions in the configuration.
So the 'variable above' is obsolete, isn't it?